deeplake/core/vectorstore/deeplake_vectorstore.py

```diff
@@ -299,7 +345,7 @@ def search(
         embedding_function: Optional[Callable] = None,
         embedding: Optional[Union[List[float], np.ndarray]] = None,
         k: int = 4,
-        distance_metric: str = "COS",
+        distance_metric: Optional[str] = None,
```

Comment:
@sounakr Cal you pls clarify the docstring since it no longed matches the code behavior. It should be clear how the distance metric selected when it is specified as None.